### PR TITLE
Remove NetFX Compression build

### PR DESCRIPTION
**Pull request by ianhays**
The cost of shipping OOB for Desktop is very high and we have burned ourselves (System.Net.Http). Let's make sure we do not accidentally run into the same problems here as well.

resolves https://github.com/dotnet/corefx/issues/15693

### CODE REVIEW

File: src/System.IO.Compression/tests/Configurations.props [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
+      netcoreapp-Unix;
```
Comment (weshaggard) [inline src/System.IO.Compression/tests/Configurations.props]: Why did you need to add the netcoreapp configurations her?

**Comment by weshaggard** [comment]
While the changes look mechanically correct I would like to get @eristj's opinion about removing this individual OOB package before we merge this PR.

**Comment by ianhays** [comment]
>I would like to get @eristj's opinion about removing this individual OOB package before we merge this PR.

Agreed.

**Comment by ianhays** [comment]
@ericstj any thoughts on this?-

**Comment by ericstj** [comment]
Please update the package index baseline to remove the non-stable info and roll it back to 4.3.0.

This isn't eliminating the OOB, its just making it so that we won't ship new versions of it in the future.  We can still update it from the servicing branch for 1.1.

@weshaggard is the 4.3.0 version of the package sufficient such that when installed in a net461 project it will fill in the netstandard2.0 surface area?

**Comment by weshaggard** [comment]
>@weshaggard is the 4.3.0 version of the package sufficient such that when installed in a net461 project it will fill in the netstandard2.0 surface area?

Yes the surface area is covered. Doing a quick diff I only see overrides as different which aren't an issue.

**Comment by ericstj** [comment]
In that case please rebase, clean up the index merge.

To fix the index change this:
https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.Private.PackageBaseline/packageIndex.json#L1722

To 4.3.0

And remove this:
https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.Private.PackageBaseline/packageIndex.json#L1748

**Comment by ianhays** [comment]
Thanks for taking a look @ericstj. I updated the PR with your fixes.
